### DISCUSSION TITLE
Taxing Speed

### DISCUSSION THREAD

**Discussion by KalTiwana** [flybywiresim/aircraft#6023] [Help] [answered]
Why is the taxing speed increasing while taxing with the throttles at idle power?  In order to control it, brakes have to be constantly applied. It is very annoying.

**Comment by tracernz** [comment, marked as answer]
That’s how a real A320neo behaves. Our goal is accuracy rather than convenience.

**Reply by KalTiwana** [reply]
Thanks for your speedy reply.  I have been in aviation all my life, but I have never experienced taxing an aircraft on brakes.  Aircraft brakes are very expensive and pilots usually avoid brakes unless absolutely necessary.  Taxing speed is set by the throttles.

**Reply by GhostEagle68** [reply]
The NEO engines are also very powerful that on idle power they can still provide enough thrust to make the plane still gather speed on the ground @KalTiwana

**Reply by Benjozork** [reply]
This has been confirmed as realistic by real-world A320neo pilots. This will not be changed.

**Comment by KalTiwana** [comment]
Thanks for your input.
